aiida/backends/tests/orm/data/test_data.py

```diff
@@ -1,72 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@sphuber Is deleting  `test_data.py` intended?
I just searched for  `validate_incoming` in this page and only found it here...

Reply:
Yes, those tests were fundamentally broken but never detected because were not run as they were not added to the list of tests to be run. In any case, they are not necessary, because what they were supposed to test is properly tests in `aiida.backends.tests.orm.node.test_node.py`

Reply:
Ok, please include this in the commit message when you squash

Reply:
Will do